core/controllers/question_editor.py

```diff
@@ -30,18 +30,23 @@ class QuestionCreationHandler(base.BaseHandler):
     """A handler that creates the question model given a question dict."""
 
     @acl_decorators.can_manage_question_skill_status
-    def post(self, comma_separated_skill_ids):
+    def post(self):
         """Handles POST requests."""
-        skill_ids = comma_separated_skill_ids.split(',')
+        skill_ids = self.payload.get('skill_ids')
+
+        if not skill_ids:
+            raise self.InvalidInputException(
+                'skill_ids parameter isn\'t present in the payload')
+
         if len(skill_ids) > constants.MAX_SKILLS_PER_QUESTION:
             raise self.InvalidInputException(
                 'More than %d QuestionSkillLinks for one question '
                 'is not supported.' % constants.MAX_SKILLS_PER_QUESTION)
         try:
             for skill_id in skill_ids:
                 skill_domain.Skill.require_valid_skill_id(skill_id)
-        except Exception:
-            raise self.InvalidInputException
+        except Exception, e:
```

Comment:
Better to do "except Exception as e": https://stackoverflow.com/a/2535770

Reply:
Done